deepchecks/nlp/checks/model_evaluation/weak_segments_performance.py

```diff
@@ -63,6 +62,10 @@ def run_logic(self, context: Context, dataset_kind) -> CheckResult:
         # Decide which scorer and score_per_sample to use in the algorithm run
         is_multilabel = text_data.is_multi_label_classification()
         if is_multilabel:
+            if self.alternative_scorer is None:
+                self.alternative_scorer = {'Precision Macro': 'precision_macro'}
```

Comment:
why not f1? precision optimizes for lowering FP while our loss is more "balanced" between FP and FN, which may create a situation in which a weak segment is detected that has a higher / equivalent precision compared to baseline, while it's loss is higher.

Reply:
Not a strong preference, changed